### PR TITLE
Additional tests for network

### PR DESCRIPTION
**Pull request by banasiakadam60**
Signed-off-by: Adam Banasiak <banasiakadam60@gmail.com>

### CODE REVIEW

File: tests/golem/network/p2p/test_node.py [hunk 1]
```diff
@@ -2,15 +2,36 @@
...
+def is_ip_address(address):
+    """
+    Check if @address is correct IP address
+    :param address: Address to be checked
+    :return: True if is correct, false otherwise
+    """
+    import socket
+    try:
+        # will raise socket.error in case of incorrect address
+        socket.inet_pton(socket.AF_INET, address)
```
Comment (badb) [inline tests/golem/network/p2p/test_node.py]: Fails on windows

> socket.inet_pton(address_family, ip_string)
> (...)
> availability: Unix (maybe not all platforms).

**Review by badb — CHANGES_REQUESTED** [review]

File: tests/golem/network/p2p/test_node.py (hunk 1 above)
Comment (mfranciszkiewicz) [inline tests/golem/network/p2p/test_node.py]: `ipaddress.ip_address` might be a good replacement

File: tests/golem/network/ipfs/test_ipfs_daemon_manager.py [hunk 2]
```diff
@@ -12,12 +12,8 @@ class TestIPFSDaemonManager(unittest.TestCase):
 
     def testStoreInfo(self):
         dm = IPFSDaemonManager(connect_to_bootstrap_nodes=False)
-        dm.store_client_info()
-
-        ipfs_id = dm.node_id
-
+        ipfs_id = dm.store_client_info()
```
Comment (mfranciszkiewicz) [inline tests/golem/network/ipfs/test_ipfs_daemon_manager.py:15]: You could also validate the value of `ipfs_id`

File: tests/golem/network/p2p/test_node.py (hunk 1 above)
Comment (mfranciszkiewicz) [reply tests/golem/network/p2p/test_node.py]: `ipaddress.ip_address`

Comment (banasiakadam60) [reply tests/golem/network/p2p/test_node.py]: Fixed

File: tests/golem/network/p2p/test_node.py [hunk 3]
```diff
@@ -11,7 +11,7 @@ def is_ip_address(address):
     import socket
     try:
         # will raise socket.error in case of incorrect address
-        socket.inet_pton(socket.AF_INET, address)
+        socket.inet_aton(address)
```
Comment (mfranciszkiewicz) [inline tests/golem/network/p2p/test_node.py]: IPv4 addresses only.

Comment (banasiakadam60) [reply tests/golem/network/p2p/test_node.py]: Done

**Review by badb — APPROVED** [review]

**Pull request merged**